src/main/java/com/github/dockerunit/discovery/consul/ConsulDescriptor.java

```diff
@@ -9,42 +18,35 @@
 import com.github.dockerunit.core.annotation.Svc;
 import com.github.dockerunit.discovery.consul.annotation.UseConsulDns;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.logging.Logger;
-
-import static com.github.dockerunit.discovery.consul.ConsulDiscoveryConfig.CONSUL_DNS_ENABLED_DEFAULT;
-import static com.github.dockerunit.discovery.consul.ConsulDiscoveryConfig.CONSUL_DNS_ENABLED_PROPERTY;
-
-@Svc(name = "consul", image = "consul:1.4.4")
+@Svc(name = "consul", image = "")
 public class ConsulDescriptor {
 
+    private static final Logger logger = Logger.getLogger(ConsulDescriptor.class.getSimpleName());
+    private static final String CONSUL_DEFAULT_IMAGE = "consul:1.4.4";
```

Comment:
As this one is now configurable, maybe it could be extracted into `ConsulDiscoveryConfig`

Reply:
:+1: